.github/workflows/pandas-tests.yaml

```diff
@@ -0,0 +1,27 @@
+name: Pandas Test Job
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        required: true
+        type: string
+      date:
+        required: true
+        type: string
+      sha:
+        required: true
+        type: string
+
+jobs:
+  pandas-tests:
+      # run the Pandas unit tests
+      secrets: inherit
+      uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@branch-24.06
+      with:
+        matrix_filter: map(select(.ARCH == "amd64" and .PY_VER == "3.9" and .CUDA_VER == "12.2.2" ))
```

Comment:
Hardcoding matrix filter values is error prone since they will fail as our support matrix changes.

What's the intended matrix that this should run against? The latest CUDA version and earliest Python version?

Once I know that information, I can help you put together a more generalized matrix filter.

cc: @bdice for additional input.

Reply:
This is because the baseline job is being run as `nightly` job and comparison job is being run as a `pull-request` job. The only matrix combination common to both is 
```
- { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04', gpu: 'v100', driver: 'latest' }
```


https://github.com/rapidsai/shared-workflows/blob/13e008c746e30a830c4ebe83a6786862858dfcb8/.github/workflows/wheels-test.yaml#L75-L94`

Reply:
@bdice too pointed out this inconsistency before, but I haven't found a way to write a matrix filter that will fetch an identical job for both the jobs of `pandas-tests` in `pr.yaml` and `branch.yaml`

Reply:
Using different job configurations is not an option here because that will create some noise(test failures) that's specific to certain configurations.

Reply:
This kind of thing is why I've advocated for the nightly matrix to be a strict superset of the PR matrix.

https://github.com/rapidsai/shared-workflows/pull/176#discussion_r1500911279
https://github.com/rapidsai/build-planning/issues/5